packages/react-components/react-button/src/components/ToggleButton/ToggleButton.types.ts

```diff
@@ -19,4 +19,8 @@ export type ToggleButtonProps = ButtonProps & {
   checked?: boolean;
 };
 
-export type ToggleButtonState = ButtonState & Required<Pick<ToggleButtonProps, 'checked'>>;
+export type ToggleButtonState = ButtonState &
+  Required<Pick<ToggleButtonProps, 'checked'>> & {
+    name: string;
+    value: string;
+  };
```

Comment:
Do those types exist on `useToggleButton_unstable`?

This type seems to be wrongly implemented. I'm assuming no errors are being screamed by typescript because of the casting provided at `useToggleState` return value, but something tells me that this is wrong! can you confirm those types @miroslavstastny

Reply:
Oh, my bad, I have put that in the wrong place, fixed here https://github.com/microsoft/fluentui/pull/24380/commits/41721b4a37e922f2425e55eb9b16db358c122225